dbfit-java/core/src/main/java/dbfit/api/vendor/DbStoredProcedureCall.java

```diff
@@ -0,0 +1,16 @@
+package dbfit.api.vendor;
```

Comment:
I'm not sure. I was thinking of something like `api.vendor` and `api.client` packages.